src/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const BreadcrumbContainer = styled.div`
+  margin: 0 auto 2rem;
+  max-width: 600px;
+  padding-bottom: 2rem;
+  @media print {
+    display: none;
+  }
+`
+
+const StepContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
+  &::before {
+    position: absolute;
+    top: 45%;
+    right: 12%;
+    left: 12%;
+    z-index: -1;
+    background-color: #c7d7df;
+    height: 10%;
+    content: '';
+  }
+`
+
+interface StepProps {
+  current: boolean
+  completed: boolean
+}
+
+const Step = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding: 0 4%;
+`
+
+const StepCircle = styled.div<StepProps>`
+  position: relative;
+  border-radius: 50%;
+  background: ${({ completed, current }) =>
+    completed ? '#ffffff' : current ? '#35809c' : '#c7d7df'};
+  width: 100%;
+  height: 0;
+  padding-top: 100%;
+  & > span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    color: ${({ completed }) => (completed ? '#35809c' : '#ffffff')};
+    font-size: 7vw;
+    font-weight: 700;
+    @media (min-width: 686px) {
+      font-size: 2rem;
+    }
+  }
+`
+
+const StepLabel = styled.div<StepProps>`
+  position: absolute;
+  margin: 0.25rem -1rem 0;
+  width: calc(100% + 2rem);
+  text-align: center;
+  color: ${({ current }) => (current ? undefined : '#c7d7df')};
+  font-weight: ${({ current }) => (current ? '600' : undefined)};
+`
+
+const Breadcrumbs = ({ step }: { step: number }) => (
+  <BreadcrumbContainer>
+    <StepContainer>
+      {['Mark', 'Review', 'Print', 'Cast'].map((label, i) => {
```

Comment:
nit, not a blocker, possible future enhancement: you've made a nice component that could be reused for any wizard type thing, but the steps are hard-wired. Feels like those should be props or children elements to `Breadcrumbs`, especially since the instructions that match the step are provided outside of this component, so there's a bit of an abstraction mismatch.

Reply:
Yeah, thought about this. The component only supports 4 steps, so I'm kinda enforcing this by hard coding the steps in. Agreed on the abstraction though. I didn't want to spend any more time making this support more than 4 steps. I guess I could have added a generic check in the component to error if more than four steps.

Reply:
Yeah, thought about this. The component only supports 4 steps, so I'm kinda enforcing this by hard coding the steps in. Agreed on the abstraction though. I didn't want to spend any more time making this support more than 4 steps. I guess I could have added a generic check in the component to error if more than four steps.